build.gradle

```diff
@@ -18,24 +19,44 @@ repositories {
     jcenter()
 }
 
+ext {
+    aspectjweaverVersion = '1.8.13'
+    testngVersion = '6.13.1'
+    seleniumJavaVersion = '3.8.1'
+    webdrivermanagerVersion = '2.0.1'
+    ownerVersion = '1.0.9'
+    lombokVersion = '1.16.18'
+    streamExVersion = '0.6.6'
+    joorVersion = '0.9.7'
```

Comment:
could you run tests to make sure it works, as in new joor version there were made some outstanding changes related to java 9 compatibility, which may break things...